clippy_lints/src/matches.rs

```diff
@@ -892,6 +925,30 @@ fn check_match_single_binding(cx: &LateContext<'_, '_>, ex: &Expr<'_>, arms: &[A
     }
 }
 
+fn check_match_refcell_borrow(cx: &LateContext<'_, '_>, expr: &Expr) {
+    match &expr.kind {
+        ExprKind::MethodCall(_, _, args) => {
+            if let [arg] = &**args {
+                let method_def_id = cx.tables.type_dependent_def_id(expr.hir_id).unwrap();
```

Comment:
Can you match this, instead of unwrapping? Unwrapping of `cx.tables` queries resulted in some ICEs in Clippy and I'd like to prevent this. If you're really sure that this won't lead to problems, please use `expect`